google/cloud/firestore_v1/async_aggregation.py

```diff
@@ -150,13 +150,13 @@ def stream(
                 to a system-specified value.
 
         Returns:
-            async_stream_iterator.AsyncStreamIterator: A generator of the query
-            results.
+            `AsyncStreamGenerator[DocumentSnapshot]`:
```

Comment:
Do you need the ` quotes here?

Reply:
Honestly I'm not super sure about this. I added the backticks because they are used in many other type annotations (for [example](https://github.com/googleapis/python-firestore/blob/104293b88cfc0af411136d87bdac67810cb938fb/google/cloud/firestore_v1/client.py#L170)). (A quick search shows the backtick makes sphinx [create a link in the docs](https://developer.lsst.io/v/u-ktl-third-party-team/python/numpydoc.html#marking-up-parameter-names), but I'm not very clear about which documentation standard we are following.) Maybe we can clean this up along with other type annotations in a future PR. WDYT?